js/packages/common/src/contexts/meta/loadAccounts.ts

```diff
@@ -514,3 +515,57 @@ export const initMetadata = async (
     }
   }
 };
+
+export const loadMultipleAccounts = async (
```

Comment:
This is a helper function that takes a list of account public keys, calls get multiple accounts, processes the account based on its owner, and returns a meta state snippet that can be merged into the current state.

Reply:
My only issue here is that there already is a helper called getMultipleAccounts in common. So may want to possibly rename this.